### PR TITLE
Expose coverage amount for employee benefit

### PR DESCRIPTION
**Pull request by gchavan**
cc @lee-at-gusto

### CODE REVIEW

File: _posts/2014-07-02-employee_benefits.markdown [hunk 1]
```diff
@@ -25,6 +25,7 @@ Employee benefits represent an <a href="/v1/employees">employee</a> enrolled in
 | `deduct_as_percentage` | Boolean | | X | false | if true, the employee_deduction amount will be treated as a percentage to be deducted from each payroll
 | `contribute_as_percentage` | Boolean | | X | false | if true, the company_contribution amount will be treated as a percentage to be deducted from each payroll
 | `catch_up` | Boolean | | X | | if true, the employee should use a benefit's special 'catch up' rate. Currently only the Roth 401k and 401k use this value for employees over 50.
+| `coverage_amount` | String | | X | null | the amount that the employee is insured for (only applicable for group term life benefit)
```
Comment (robbiegill) [inline _posts/2014-07-02-employee_benefits.markdown]: Can you add an extra note that even for GTL you cannot send a non-blank value for both this and company contribution?

Comment (gchavan) [inline _posts/2014-07-02-employee_benefits.markdown]: certainly

**Comment by robbiegill** [comment]
@alfred accept!

**Comment by zp-alfred** [comment]
Accepted by robbiegill!

**Comment by gchavan** [comment]
@robbiegill how do I land this? should i just merge?

**Comment by robbiegill** [comment]
Yep.

**Pull request merged**